components/permit-holders/PermitHolderHeader.tsx

```diff
@@ -24,15 +25,23 @@ export default function PermitHolderHeader({
       <Flex marginTop={5} alignItems="center">
         <Box>
           <Flex alignItems="center">
-            <Text textStyle="display-large" as="h1" marginRight={3}>
-              {`${applicant.firstName} ${applicant.lastName}`}
-            </Text>
-            <RequestStatusBadge variant={applicationStatus} />
+            {applicant && (
+              <Text textStyle="display-large" as="h1" marginRight={3}>
+                {`${applicant.firstName} ${applicant.lastName}`}
+              </Text>
+            )}
+            {applicantStatus && (
+              <Wrap>
+                <Badge variant={applicantStatus}>{applicantStatus}</Badge>
```

Comment:
Please use `RequestStatusBadge` here. We have a backlog ticket that will separate request and applicant statuses: https://www.notion.so/uwblueprintexecs/Separate-RequestStatusBadge-from-PermitStatusBadge-abd47cbefc204a2cae28e13cab7fcc5f

Reply:
@OustanDing The `RequestStatusBadge` doesn't support `INVALID` types so if the applicant status is invalid then no badge will render, is this what you want? Also to add, in the permit-holders table the `Badge` which is being used is the same one that we are using

Reply:
Oh I see. Okay, we can leave it for now then and look more into in the near future.